src/main/java/seedu/address/logic/parser/PomCommandParser.java

```diff
@@ -14,10 +14,6 @@ public PomCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_TIMER);
```

Comment:
should throw an error if <= 0. Actually might want to raise the minimum to 0.5? (feature decision, u can decide, but at least make it <= 0, pom 0 shouldn't happen)

Reply:
pom 0 as in you mean the task index?